devices/GAT-123/out/metadata_norm.json

```diff
@@ -23,6 +23,12 @@
   },
   "pointset": {
     "points": {
+      "filter_alarm_pressure_status": {
```

Comment:
At the end of this (using the site defaults file) -- I think for most of these there should be 0 change in the generated out -- it's just that the value is supplied by the defaults but not the device metadata file... There might be a few of these (metadata_norm.json) files changed, but only as a "proof" that something about the defaults is working as intended (e.g. something overwritten by a device-level field).